docs/source/quickstart.rst

```diff
@@ -6,24 +6,37 @@ Quickstart
 Installing MLflow
 -----------------
 
-You install MLflow by running:
+You install MLflow with extra dependencies by running:
```

Comment:
Why do we need to make this distinction in our Quickstart? The Quickstart should be simple and minimally confusing. Just say what to run (maybe explain that extra means "with some additional dependencies").

Reply:
Suggested in discussion with @dbczumar.